src/screens/FeedScreen/hooks/useCoreFeed.js

```diff
@@ -35,6 +29,12 @@ const useCoreFeed = () => {
   const {myProfile} = profileContext;
   const {bottom} = useSafeAreaInsets();
 
+  const {sendViewPostTimeWithFeeds, updateViewPostTime, isSamePostViewed} = useViewPostTimeHook(
+    dispatch,
+    timer,
+    viewPostTimeIndex
+  );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The usage of `useViewPostTimeHook` here is correct. It's good that you are passing necessary dependencies (`dispatch`, `timer`, `viewPostTimeIndex`) to the hook.
```diff
+ const {sendViewPostTimeWithFeeds, updateViewPostTime, isSamePostViewed} = useViewPostTimeHook(
+   dispatch,
+   timer,
+   viewPostTimeIndex
+ );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
closed